include/libbase64.h

```diff
@@ -18,6 +18,7 @@ extern "C" {
 
 struct base64_state {
 	int eof;
+	int aeof;
```

Comment:
Indeed. The caller is free to allocate the structure as he pleases, including on the stack. I wanted to avoid all dynamic memory troubles.